compiler/damlc/daml-opts/daml-opts-types/DA/Daml/Options/Types.hs

```diff
@@ -157,6 +170,7 @@ mkOptions opts@Options {..} = do
     ghcVersionFile <- locateRunfiles (mainWorkspace </> "compiler" </> "damlc" </> "ghcversion.h")
 
     pure opts {
+        optPackageImports = optPackageImports ++ basePackageImports optMbPackageName,
```

Comment:
Instead of injecting the base imports here, I would suggest to do this in `initPackageDb` if `isProject` returns `False`.

Reply:
The base import injection is now done in: 
`compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs`